### DISCUSSION TITLE
PageImages for mode=gallery?

### DISCUSSION THREAD

**Discussion by NeilanNaicker** [Universal-Omega/DynamicPageList4#26] [Q&A]
DynamicPageList (WikiMedia) has a feature whereby pages requested in mode=gallery don't need to be File: pages, as long as they have a PageImage (via the PageImages extension). Is there some similar means to pull an image from a page in order to create a gallery of standard pages?

**Comment by Universal-Omega** [comment]
Hey, I appreciate you making use of this "Discussions" feature for this rather then the issues. Please know I am not the original developer of this extension so I don't know everything about this extension, but I do not believe this is a possibility at this time, with DynamicPageList3, but I can try to implement such a feature in the future.
